packages/aragon-api/src/index.js

```diff
@@ -360,6 +363,10 @@ export class AppProxy {
 
         return getPastEvents(cachedBlock, pastEventsToBlock).pipe(
           mergeScan(wrappedReducer, { ...cachedState, ...initState }, 1),
+          tap((state) => {
```

Comment:
I would put this after the `last` operator. The pastEvents come all at once, therefor I'd first reduce them all and then render. This could make it a little sluggish.

Reply:
Opted to throttle these updates with a `sampleTime()`